howitworks.md

```diff
@@ -6,6 +6,18 @@ top_graphic: 3
 redirect_from: "/howitworks"
 ---
 
+# What is Let's Encrypt?
+
+Let's Encrypt allows you to secure your website by concealing the data
+sent and received between your website and the users of your website from prying
+eyes. Prior to the creation of Let's Encrypt, securing your website with SSL/HTTPS
```

Comment:
I would just use HTTPS here (see above for suggestion about incorporating "SSL" for people looking for that acronym).